lib/src/test/java/com/uber/cherami/client/CheramiIntegrationTest.java

```diff
@@ -1,302 +0,0 @@
-/*******************************************************************************
```

Comment:
Is this removed because we no longer need it? or will it be added later once we have a proper cherami-server mock?